src/compositemark/callout.ts

```diff
@@ -39,12 +34,16 @@ export interface CalloutConfigMixins {
    * @hide
    */
   callout?: CalloutConfig;
+  calloutLabel?: MarkConfig;
```

Comment:
let's remove `config.calloutLabel`, `config.calloutRule` and make `config.callout.label/rule`.